src/Workspaces/VisualBasic/Portable/CodeCleanup/Providers/ReduceTokensCodeCleanupProvider.vb

```diff
@@ -88,8 +90,6 @@ Namespace Microsoft.CodeAnalysis.CodeCleanup.Providers
                         End If
 
                         Dim base = literal.GetBase()
-
-                        Const digitSeperator = "_"c
                         If Not base.HasValue OrElse idText.Contains(digitSeperator) Then
```

Comment:
why do we need to do the check again?

Reply:
Nope, just forgot to remove it when I move the check to pre `Select Case`